lib/batch/index.js

```diff
@@ -12,8 +12,8 @@ function Batch(options) {
 	this._entries = [];
 
 	// Allow the batch header/control defaults to be overriden if provided
-	this.header = options.header ? _.merge(options.header, require('./header'), _.defaults) : _.cloneDeep(require('./header'));
-	this.control = options.control ? _.merge(options.header, require('./control'), _.defaults) : _.cloneDeep(require('./control'));
```

Comment:
this line was checking for `options.control` but loading `options.header` if it was set.